sycl/CMakeLists.txt

```diff
@@ -176,7 +176,7 @@ if ("opencl" IN_LIST SYCL_ENABLE_BACKENDS)
   set(SYCL_BUILD_BACKEND_OPENCL ON)
 endif()
 if ("level_zero" IN_LIST SYCL_ENABLE_BACKENDS)
-  set(SYCL_BUILD_BACKENDLEVEL_ZERO ON)
+  set(SYCL_BUILD_BACKEND_LEVEL_ZERO ON)
```

Comment:
If I am not mistaken the only usage of this variable is in LIT to enable level_zero feature. I wonder how it even worked with this typo and whether we were just skipping tests which had level_zero requirement.